packages/button/src/css/index.ts

```diff
@@ -74,79 +71,84 @@ export default {
   },
   [`.psds-button--size-${vars.sizes.xSmall}`]: {
     fontSize: type.fontSize100,
+    letterSpacing: type.letterSpacingLooser,
     padding: `0 ${layout.spacingXSmall}`,
     height: '24px'
   },
   [`.psds-button--size-${vars.sizes.small}`]: {
     fontSize: type.fontSize200,
-    padding: `${layout.spacingXXSmall} ${layout.spacingXSmall}`,
+    letterSpacing: type.letterSpacingLooser,
+    padding: `${layout.spacingXXSmall} ${layout.spacingSmall}`,
     height: '32px'
   },
   [`.psds-button--size-${vars.sizes.medium}`]: {
-    fontSize: type.fontSize200,
+    fontSize: type.fontSize300,
+    letterSpacing: type.letterSpacingLoose,
     padding: `${layout.spacingXXSmall} ${layout.spacingMedium}`,
     height: '40px'
   },
   [`.psds-button--size-${vars.sizes.large}`]: {
     fontSize: type.fontSize400,
-    padding: `${layout.spacingXXSmall} ${layout.spacingMedium}`,
+    letterSpacing: type.letterSpacingNone,
+    padding: `${layout.spacingXXSmall} ${layout.spacingLarge}`,
     height: '48px'
   },
   [`.psds-button--appearance-${vars.appearances.primary}.psds-theme--${themeDefaultName}`]: {
     color: colorsTextIcon.highOnDark,
     background: colorsPrimaryAction.background,
     '&:not([disabled]):hover': {
-      background: colorsBlue[7],
+      background: colorsBlue[8],
       cursor: 'pointer'
     },
     '&:not([disabled]):focus': {
-      background: colorsBlue[7],
+      background: colorsBlue[8],
       cursor: 'pointer'
     },
     '&:not([disabled]):active': {
-      background: colorsBlue[8]
+      background: colorsBlue[9]
     }
   },
   [`.psds-button--appearance-${vars.appearances.primary}.psds-theme--${themeNames.light}`]: {
     color: colorsTextIcon.highOnDark,
     background: colorsPrimaryAction.background,
     '&:not([disabled]):hover': {
-      background: colorsBlue[7],
+      background: colorsBlue[8],
```

Comment:
Thanks for tweaking all these.  I'd love to have worked with you to see your process in a11y color checking all these.